tests/album-create.test.js

```diff
@@ -0,0 +1,48 @@
+// tests/album-create.js
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  //first create a new artist to use in album table
```

Comment:
Good to treat comments like real sentences and use the same structure
```suggestion
  // First create a new artist to use in album table.
```